portfolio/src/main/java/com/google/sps/data/CommentListSingleton.java

```diff
@@ -0,0 +1,53 @@
+package com.google.sps.data;
+
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.HashMap;
+
+import com.google.sps.dataExceptions.CommentExistingId;
+import com.google.sps.dataExceptions.CommentNotFound;
+
+public class CommentListSingleton {
+    private static CommentListSingleton instance;
+
+    public static CommentListSingleton getInstance() {
+        if(instance == null) {
+            instance = new CommentListSingleton();
+        }
```

Comment:
Missed it before. This code is not thread safe. I believe accepted answer on the following page shows correct way of implementing singleton with lasy initialisation - https://stackoverflow.com/questions/16106260/thread-safe-singleton-class

Reply:
Done